ostree-compliance-mode.spec

```diff
@@ -0,0 +1,37 @@
+Name:           ostree-compliance-mode
+Version:        0.1
+Release:        1%{?dist}
+Summary:        ostree-compliance-mode
```

Comment:
We need to write a summary;

`ostree-compliance-mode is a tool designed to purge protected data, leaving the system in a modifyable state in compliance with GPLv3.`